src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -101,6 +101,26 @@ private void PrintHelp()
       bicep decompile file.json --outdir dir1
       bicep decompile file.json --force
       bicep decompile file.json --outfile file.bicep
+
+  {exeName} generate-params [options] <file>
+    Builds .parameters.json file from the given bicep file.
+
+    Arguments:
+      <file>        The input file
+
+    Options:
+      --no-restore      Generates the parameters file without restoring external modules.
+      --outdir <dir>    Saves the output at the specified directory.
+      --outfile <file>  Saves the output as the specified file path.
+      --stdout          Prints the output to stdout.
+
+    Examples:
+      bicep generate-params file.bicep
+      bicep generate-params file.bicep --no-restore
+      bicep generate-params file.bicep --stdout
+      bicep generate-params file.bicep --outdir dir1
+      bicep generate-params file.bicep --outfile file.parameters.json
+
```

Comment:
@ucheNkadiCode Would it be better to have a specific option to do merging of an existing params file?  E.g. bicep generate-params file.bicep --update

Reply:
if I remember correctly, the consensus was that users are already using source control mechanism, if somehow the end result is not what they want, they can use git techniques to revert changes. to keep simplicity of the code and behavior, it should always behave the same way.

Reply:
@shenglol, @anthony-c-martin , @majastrz do you have an opinion here? My thoughts are if the file exists and they type this command into cli, it would make sense to overwrite it here